frontend/src/main/main.js

```diff
@@ -163,10 +186,9 @@ class Viz extends React.Component {
         )
     }
 }
-// data: PropTypes.arrayOf(PropTypes.object).isRequired,
 
 Viz.propTypes = {
-    data: PropTypes.objectOf(PropTypes.array).isRequired,
+    data: PropTypes.object.isRequired, //Ask Stefan about interior contents (objs or arrays)
```

Comment:
As discussed, yes, you are correct, data is no longer an object of arrays but rather just an object (it contains both arrays and objects). This change solves the error message.